abci/example/kvstore/README.md

```diff
@@ -1,24 +1,10 @@
 # KVStore
 
-There are two app's here: the KVStoreApplication and the PersistentKVStoreApplication.
-
-## KVStoreApplication
-
 The KVStoreApplication is a simple merkle key-value store. 
 Transactions of the form `key=value` are stored as key-value pairs in the tree.
 Transactions without an `=` sign set the value to the key.
 The app has no replay protection (other than what the mempool provides).
 
-## PersistentKVStoreApplication
```

Comment:
Here as well, my question is: what was, technically, what forced you to refactor the KV application  **in this diff**?